hack/jenkins/benchmark-dockerized.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+retry() {
+  for i in {1..5}; do
+    "$@" && return 0 || sleep $i
+  done
+  "$@"
+}
+
+# Runs benchmark integration tests, producing JUnit-style XML test
+# reports in ${WORKSPACE}/artifacts. This script is intended to be run from
+# kubekins-test container with a kubernetes repo mapped in. See
+# k8s.io/test-infra/scenarios/kubernetes_verify.py
+
+export PATH=${GOPATH}/bin:${PWD}/third_party/etcd:/usr/local/go/bin:${PATH}
+
+retry go get github.com/tools/godep && godep version
+retry go get github.com/jstemmer/go-junit-report
+
+# Disable the Go race detector.
+export KUBE_RACE=" "
+# Disable coverage report
+export KUBE_COVER="n"
```

Comment:
Isn't that default?

Reply:
I guess so (as it seems like we'll reach this codepoint https://github.com/kubernetes/kubernetes/blob/master/hack/make-rules/test.sh#L102).
But I just set it anyway to make it explicit here and be on the safe side. We are doing it here too - https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/test-dockerized.sh

Reply:
I can remove it if you want.